MONGODB/F1.2.md

```diff
@@ -0,0 +1,37 @@
+# F1.1 An Introduction to MongoDB and Atlas
```

Comment:
All your image tags are local. You need  to upload them imgur or something and replace the links.